website/public/js/controllers/challengesCtrl.js

```diff
@@ -116,11 +114,14 @@ habitrpg.controller("ChallengesCtrl", ['$rootScope','$scope', 'Shared', 'User',
         rewards: clonedTasks.reward,
         leader: User.user._id,
         group: challenge.group._id,
-        timestamp: +(new Date),
-        members: [],
         official: challenge.official,
         prize: challenge.prize
       });
+
+      function _cloneTaskAndPush(taskToClone) {
```

Comment:
Made this a private method. Since the second and third arguments aren't referenced, we didn't need to include them.